enterprise/backend/src/metabase_enterprise/transforms/models/transform_job.clj

```diff
@@ -90,3 +92,38 @@
                         {:job_id   job-id
                          :tag_id   tag-id
                          :position (get new-positions tag-id)})))))))
+
+(defn localize-jobs
+  "Localize the initial jobs in the transform_job table."
+  []
+  (let [un-localized (t2/select :model/TransformJob :built_in_type [:is-not nil])]
+    (when (seq un-localized)
+      (log/info (str "Localizing jobs for site locale."))
+      (let [values {"hourly"
+                    [(i18n/trs "Hourly job")
+                     (i18n/trs "Executes transforms tagged with ''hourly'' every hour")]
+
+                    "daily"
+                    [(i18n/trs "Daily job")
+                     (i18n/trs "Executes transforms tagged with ''daily'' once per day")]
+
+                    "weekly"
+                    [(i18n/trs "Weekly job")
+                     (i18n/trs "Executes transforms tagged with ''weekly'' once per week")]
+
+                    "monthly"
+                    [(i18n/trs "Monthly job")
+                     (i18n/trs "Executes transforms tagged with ''monthly'' once per month")]}]
+        (doseq [{:keys [built_in_type]} un-localized
+                :let [[name description] (get values built_in_type)]
+                :when name
+                :when (pos? (t2/update! :model/TransformJob
+                                        :built_in_type built_in_type
+                                        {:name name
+                                         :description description
+                                         :built_in_type nil}))]
+          (log/info (str "Localized " built_in_type " job for site locale.")))))))
+
+(defmethod task/init! ::LocalizeJobs [_]
+  (when (setup/has-user-setup)
```

Comment:
this doesn't appear to be a one time thing, but every time we startup?

Reply:
Yes, it's every time we start. But it should only ever change anything once.